### DISCUSSION TITLE
Cloudfront videos not downloaded

### DISCUSSION THREAD

**Discussion by Amazik** [aclap-dev/video-downloadhelper#533] [Help and Q&A]
### Your question

Cloudfront videos not downloaded  via extension

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://learn.nwkings.com/s/courses/5e4a65ec0cf292c0c53fb975/take",
  "error": {
    "details": "SideDownload error: : [hls @ 000002a1ac1079c0] Error when loading first segment 'https://d2a5xnk4s7n8a6.cloudfront.net/spees/w/o/5b0ef094e4b02ac2188be0c7/v/5e7fb14f0cf25ef369f2a42e/u/5cfb84f8e4b0e9c3a26d92b6/t/356d32a98ee1f4bffbec3f96aea5a6a6/p/assets/videos/5b0ef094e4b02ac2188be0c7/2020/03/28/5e7fb14f0cf25ef369f2a42e/hls_1M_000.ts'\r\n[in#0 @ 000002a1ac107840] Error opening input: Invalid data found when processing input\r\nError opening input file https://d2a5xnk4s7n8a6.cloudfront.net/spees/w/o/5b0ef094e4b02ac2188be0c7/v/5e7fb14f0cf25ef369f2a42e/u/5cfb84f8e4b0e9c3a26d92b6/t/356d32a98ee1f4bffbec3f96aea5a6a6/p/assets/videos/5b0ef094e4b02ac2188be0c7/2020/03/28/5e7fb14f0cf25ef369f2a42e/hls_1M_.m3u8.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_8441411368563117",
    "error": "coapp_failure",
    "id": "coapp_failure_f2235dac-36ad-4b11-a2b2-e274388f2029"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user, like you.

What I just posted over here #530 might be relevant here.  You should check it out.
